nextmv/nextmv/default_app/README.md

```diff
@@ -0,0 +1,17 @@
+# Nextmv Application
+
+This is the basic structure of a Nextmv application.
+
+```text
+├── app.yaml
+├── README.md
+├── requirements.txt
+└── src
```

Comment:
This app structure doesn't work without the _main.py_, right? Am I missing something? Or, is this already preparation work intended to be used with custom entry point control? :thinking: